src/main/java/kg/jarkyn/server/Resource/ResourceReader.java

```diff
@@ -0,0 +1,23 @@
+package kg.jarkyn.server.Resource;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.Arrays;
+import java.util.List;
+
+public class ResourceReader {
+
+    public static byte[] readFile(String filePath) {
+        try {
+            return Files.readAllBytes((new File(filePath)).toPath());
+        } catch (IOException e) {
```

Comment:
I like that you're suppressing this checked exception here. However returning an empty byte array may get confused with reading an empty file. In Java 7 (or 8, can't remember which), there is something called `UncheckedIOException` that you can wrap a checked exception in and throw that instead, which you could then deal with later if you needed to.